wheel/Cargo.toml

```diff
@@ -9,6 +9,9 @@ homepage = "https://github.com/Chia-Network/clvm_rs/"
 repository = "https://github.com/Chia-Network/clvm_rs/"
 readme = "README.md"
 
+[workspace]
+members = ["."]
```

Comment:
my understanding is that this will prevent `wheel` to be included in the main workspace.
The name of this commit moves the wheel project into its *own* workspace, which contradicts the PR title.